src/simplay/src/simplay/components.py

```diff
@@ -140,6 +196,10 @@ def __init__(
         capacity: ContainerAmount = float("inf"),
         init: ContainerAmount = 0,
     ):
+        if not isinstance(capacity, (int, float)):
```

Comment:
Why can capacity in this case be an int or a float and in the case above not: https://github.com/team-simplay/SimPlay/pull/6/files#diff-1387c3c1ae83ff126a45a36b29d94c2943eee63d5ab3b01116595811f3755482R61? 
Can we align that?

Reply:
No we cannot align them. The resource capacity always is a whole unit, i.e. if the resource is a person, you can't have a fraction of a person. The container capacity might be a fraction, i.e. your boiler might store exactly 312.7 liters of water. These constraints are inherited from SimPy.